ftw/dashboard/portlets/favourites/profiles/default/actions.xml

```diff
@@ -7,7 +7,7 @@
    <object name="addtofavourites" meta_type="CMF Action" i18n:domain="ftw.dashboard.portlets.favourites">
     <property name="title" i18n:translate="">Add to Favourites</property>
     <property name="description" i18n:translate="">Add to your Favourites.</property>
-    <property name="url_expr">string:${object_url}/@@add_to_favourites</property>
+    <property name="url_expr">string:${object_url}/@@add_to_favourites?_authenticator=${object/@@authenticator/token}</property>
```

Comment:
@jone 
The [`AuthenticatorView`](https://github.com/plone/plone.protect/blob/master/plone/protect/authenticator.py#L92) provides the `token` method only since version 3.x. Any Idea how I could support both versions?